### PR TITLE
OCM-5963 | fix: add more instruction in billing account error

### PR DESCRIPTION
**Pull request by chenz4027**
https://issues.redhat.com/browse/OCM-5963

### CODE REVIEW

File: cmd/create/cluster/cmd.go [hunk 1]
```diff
@@ -1068,7 +1070,8 @@ func run(cmd *cobra.Command, _ []string) {
 						"To use a different billing account, add --billing-account xxxxxxxxxx to previous command",
 					)
 				} else {
-					r.Reporter.Errorf("A billing account is required for Hosted Control Plane clusters.")
+					r.Reporter.Errorf("A billing account is required for Hosted Control Plane clusters." +
```
Comment (mikeshng) [inline cmd/create/cluster/cmd.go:1073]:
```suggestion
					r.Reporter.Errorf("A billing account is required for Hosted Control Plane clusters. %s", listBillingAccountMessage)
```

File: cmd/create/cluster/cmd.go [hunk 2]
```diff
@@ -80,7 +80,8 @@ const (
 	MinReplicasSingleAZ = 2
 	MinReplicaMultiAZ   = 3
 
-	listInputMessage = "Format should be a comma-separated list."
+	listInputMessage          = "Format should be a comma-separated list."
+	listBillingAccountMessage = " To see the list of billing account options, you can use interactive mode by passing '-i'"
```
Comment (mikeshng) [inline cmd/create/cluster/cmd.go:84]:
```suggestion
	listBillingAccountMessage = "To see the list of billing account options, you can use interactive mode by passing '-i'"
```

minor nit. It's kind of strange to have the space in the beginning, probably better to have the space in the places where it needs to use it.

File: cmd/create/cluster/cmd.go [hunk 3]
```diff
@@ -1046,8 +1047,9 @@ func run(cmd *cobra.Command, _ []string) {
 
 		if !isHcpBillingTechPreview {
 
-			if billingAccount != "" && !ocm.IsValidAWSAccount(billingAccount) {
-				r.Reporter.Errorf("Billing account is invalid. Run the command again with a valid billing account.")
+			err := validateBillingAccount(billingAccount)
+			if err != nil {
+				r.Reporter.Errorf("%s", err)
```
Comment (mikeshng) [inline cmd/create/cluster/cmd.go:1052]:
```suggestion
				r.Reporter.Errorf("%v", err)
```

`err` is not a string right? so I think you have to use `err.Error()` or `%v`

File: cmd/create/cluster/cmd.go [hunk 4]
```diff
@@ -3253,6 +3257,14 @@ func clusterConfigFor(
 	return clusterConfig, nil
 }
 
+func validateBillingAccount(billingAccount string) error {
+	if billingAccount != "" && !ocm.IsValidAWSAccount(billingAccount) {
+		return fmt.Errorf("Billing account is invalid. Run the command again with a valid billing account." +
```
Comment (mikeshng) [inline cmd/create/cluster/cmd.go:3262]: I think it's better to use `%s` like in the above comment.

File: cmd/create/cluster/cmd.go (hunk 3 above)
Comment (chenz4027) [reply cmd/create/cluster/cmd.go:1052]: @mikeshng err will be a string as it's the return of validateBillingAccount in line 3260, but I'll fix the rest.

**Review by mikeshng — APPROVED** [review]
/lgtm

File: cmd/create/cluster/cmd.go [hunk 5]
```diff
@@ -1090,8 +1092,9 @@ func run(cmd *cobra.Command, _ []string) {
 					billingAccount = aws.ParseOption(billingAccount)
 				}
 
-				if billingAccount == "" || !ocm.IsValidAWSAccount(billingAccount) {
```
Comment (oriAdler) [inline cmd/create/cluster/cmd.go:1093]: In the function `validateBillingAccount` the condition is different, is that expected?

Comment (mikeshng) [reply cmd/create/cluster/cmd.go:1093]: Nice catch! Test case should probably cover the empty case as well.

Comment (chenz4027) [reply cmd/create/cluster/cmd.go:1093]: Nice catch @oriAdler

File: cmd/create/cluster/cmd.go [hunk 6]
```diff
@@ -80,7 +80,8 @@ const (
 	MinReplicasSingleAZ = 2
 	MinReplicaMultiAZ   = 3
 
-	listInputMessage = "Format should be a comma-separated list."
+	listInputMessage          = "Format should be a comma-separated list."
+	listBillingAccountMessage = "To see the list of billing account options, you can use interactive mode by passing '-i'"
```
Comment (oriAdler) [inline cmd/create/cluster/cmd.go:84]:
```suggestion
	listBillingAccountMessage = "To see the list of billing account options, you can use interactive mode by passing '-i'."
```

**Review by oriAdler — COMMENTED** [review]
Left a minor comment, other than that LGTM.

File: cmd/create/cluster/cmd.go (hunk 6 above)
Comment (chenz4027) [reply cmd/create/cluster/cmd.go:84]: Thanks @oriAdler , will make the change

**Comment by chenz4027** [comment]
@oriAdler Not sure why it didn't tag the labels for "lgtm" and "approved" automatically 🤔

**Comment by oriAdler** [comment]
> @oriAdler Not sure why it didn't tag the labels for "lgtm" and "approved" automatically 🤔

Maybe I had to add `lgtm` after the pipeline passed, added.

**Pull request merged**